apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -619,6 +693,49 @@ const WorkshopForm = React.createClass({
           <Row>
             <Col sm={10}>
               <FormGroup>
+                <ControlLabel>
+                  Workshop Type Options
+                  &nbsp;<a onClick={this.toggleTypeOptionsHelpDisplay}>(help)</a>
```

Comment:
I thought it would, but it wasn't; could be that the `ControlLabel` component is doing something weird